src/main/java/com/keeper/homepage/domain/member/entity/Member.java

```diff
@@ -401,4 +402,13 @@ public List<String> getJobs() {
   public void updateType(MemberType memberType) {
     this.memberType = memberType;
   }
+
+  public boolean isBorrowBooks() {
+    return !this.bookBorrowInfos.isEmpty();
+  }
```

Comment:
`is`보단 `has`가 어울릴 것 같고, `bookBorrowInfo`가 로그 성질을 가지고 있어서 반납을 완료했더라도 "반납완료"로 `bookBorrowInfos`를 가지고 있습니다. 그래서 아래 suggestion 처럼 해주어야 실제로 빌리고 있는 책들이 있는지 확인할 수 있습니다.

```suggestion
  public boolean hasAnyBorrowBooks() {
    return this.bookBorrowInfos.stream()
        .anyMatch(BookBorrowInfo::isInBorrowing);
  }
```

Reply:
수정했습니다~! suggestion 감사합니다!!